migtests/tests/oracle/datatypes/validate

```diff
@@ -22,6 +22,30 @@ EXPECTED_ROW_COUNT = {
 if os.environ.get('BETA_FAST_DATA_EXPORT') == '1':
 	EXPECTED_ROW_COUNT['long_type'] = 2 # https://yugabyte.atlassian.net/browse/DB-5929	
 
+EXPECTED_SUM_OF_COLUMN = {
+	'numeric_types': {
+		'num_val': '260.580264',
+		'float_val': '236.0018',
+		'bin_float_val': '1.361127976e+29',
+		'bin_double_val': '5.39307940458693e+108'
+	},
+	'number_ps': {
+		'num_val': '123.89',
+		'num_3': '402',
+		'num_p_s': '5.79',
+		'num_s_p': '0.0000012',
+		'num_neg_s': '546.99'
+	}
+}
+if os.environ.get('BETA_FAST_DATA_EXPORT') == '1':
```

Comment:
Why is this different for debezium?

Reply:
EXPECTED_SUM_OF_COLUMN[numeric_types][bin_float_val]

Reply:
Can you pls add a comment explaining why we're doing this, and why only this column? We'll forget later :D

Reply:
Also, we need to add asserts for table `number_ps` as well. 
(It's okay if you want to do it in a separate PR, in which case add a TODO comment here)